protocol/src/main/protobuf/UserBitShared.proto

```diff
@@ -320,3 +320,17 @@ message Jar {
   repeated string function_signature = 2;
 }
 
+
+/* Session parameter
+message SessionParam {
+  optional string key = 1;
+  optional string value = 2;
+}
```

Comment:
Though they have same key-value pairs, did not want to overload it for different purposes.